lib/reporters/junit/index.js

```diff
@@ -20,12 +20,15 @@ JunitReporter = function (newman, reporterOptions) {
             root,
             testSuitesExecutionTime = 0,
             executionTime = 0,
-            timestamp;
+            timestamp,
+            classname = _.camelCase(collection.name).replace(/\W/g, '');
```

Comment:
@donotello This statement is redundant, see the comment below.

Reply:
Rewritten, thank you.